mypyc/primitives/weakref_ops.py

```diff
@@ -0,0 +1,38 @@
+from mypyc.ir.rtypes import object_rprimitive
+from mypyc.primitives.registry import function_op
+
+# Weakref operations
+
+"""
+py_new_weak_ref_op = function_op(
+    name="weakref.weakref",
+    arg_types=[object_rprimitive],
+    # TODO: how do I pass NULL as the 2nd arg? 
```

Comment:
@JukkaL I've also added 2 new primitives for weakref.ref and weakref.proxy, when called with 2 args. I'm now working on 2 more primitives when called with 1 arg. what would you advise here?